python/ray/data/tests/test_dataset.py

```diff
@@ -3566,6 +3566,40 @@ def get_node_id():
     assert set(locations) == {node1_id, node2_id}
 
 
+def test_random_sample():
```

Comment:
```suggestion
def test_random_sample(ray_start_regular_shared):
```

(This seems to be causing a subsequent test to be failing--- maybe also need to move this test to be near the other tests that use the ray_start_regular_shared fixture).